test/views/EstablishedCountryViewSpec.scala

```diff
@@ -63,4 +66,13 @@ class EstablishedCountryViewSpec extends StringViewBehaviours {
       }
     }
   }
+
```

Comment:
This whole spoke is already behind the toggle. So there is no value in adding tests when the toggle is disabled since this page won't appear in that case